test/unit/storage-manager.spec.ts

```diff
@@ -30,6 +30,7 @@ test.group('Storage Manager', (group) => {
 	test('throw exception when disk config doesnt have driver', (assert) => {
 		const storageManager = new StorageManager({
 			default: 'local',
+			// @ts-ignore
```

Comment:
for some reason, it fails here with `npm run test` and below in the IDE. Might be because the TS version is different